core/primitives/src/sharding/shard_chunk_header_inner.rs

```diff
@@ -7,12 +7,17 @@ use near_primitives_core::hash::CryptoHash;
 use near_primitives_core::types::{Balance, BlockHeight, Gas, ShardId};
 use near_schema_checker_lib::ProtocolSchema;
 
+// When removing CryptoHash fields from new versions to be safe we return defaults instead of
+// panicking.
+const DEFAULT_CRYPTO_HASH: &CryptoHash = &CryptoHash::new();
+
 #[derive(BorshSerialize, BorshDeserialize, Clone, PartialEq, Eq, Debug, ProtocolSchema)]
 pub enum ShardChunkHeaderInner {
     V1(ShardChunkHeaderInnerV1),
     V2(ShardChunkHeaderInnerV2),
     V3(ShardChunkHeaderInnerV3),
     V4(ShardChunkHeaderInnerV4),
+    V5(ShardChunkHeaderInnerV5SpiceTxOnly),
```

Comment:
let's just keep the name as ShardChunkHeaderInnerV5? We can add more details about Spice in the description

Reply:
Unless, is there's some very specific reason for this nomenclature?

Reply:
Could you elaborate on why? I think having spice in the name makes it easier to spot (no need to look up the description). IIUC it's possible to rename it later if needed as well.